docs/users.rst

```diff
@@ -636,6 +636,49 @@ Copy the osmosis command generated by conversion tool and run from the command l
 * Environment variables: PWD=/path/to/beam/folder
 
 
+11. Warmstart
+
+We can configure beam to run from previous state. This configuration can be controlled by following parameters in config file::
+
+ * beam.warmStart.type = "disabled"
+ * beam.warmStart.path = "https://s3.us-east-2.amazonaws.com/beam-outputs/run140-base__2018-06-26_22-20-49_28e81b6d.zip"
+ * beam.warmStart.samplePopulationIntegerFlag = 0
+ * beam.warmStart.prepareData = false
+ * beam.warmStart.skimsFilePaths = [
+                                     {
+                                         skimType = "String"
+                                         skimsFilePath = ""
+                                     }
+                                   ]
+
+
+
+We can configure each param as following
+
+- `beam.warmStart.type`: This parameter control type of warm start. We have following options for this param.
+
+ - **disabled** - completely disable warm start
+ - **full** - All warm start files has been used for initialising warm start
+ - **linkStatsOnly** - only link stats is loaded (all the other data is got from the input directory)
+ - **linkStatsFromLastRun** - only link stats is loaded from beam.input.lastBaseOutputDir directory
+
+- `beam.warmStart.path`: This property consist path of warm starts files from some previous run. We can give path of zip file on s3 or local dir
+
+- `beam.warmStart.samplePopulationIntegerFlag`: This parameter control if population sampling needed or not. For full warmstart run we can not change sampling
```

Comment:
```suggestion
- `beam.warmStart.samplePopulationIntegerFlag`: This parameter controls if population sampling is needed or not. For full warmstart this will be ignored, as sampling cannot be changed.
```